thorvg/Kconfig

```
@@ -0,0 +1,55 @@
+menu "ThorVG Support Options"
+
+    config THORVG_LOG_ENABLED
+        bool "Enable ThorVG log"
+        default n
+        help
+            Enable ThorVG log.
+
+    config THORVG_THREAD_ENABLED
+        bool "Enable ThorVG thread"
```

```suggestion
        bool "Enable ThorVG threading support"
```

(otherwise it sounds like ThorVG will create an actual thread, which isn't the case I guess?)